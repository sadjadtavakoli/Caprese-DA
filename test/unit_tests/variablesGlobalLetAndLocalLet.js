let variable = 1

function big(){
    function caller1(){
        let variable = 2
        let var1 = variable
    }

    caller1()
}

function big2(){

    function caller3(){
        let var1 = variable
        variable = 5
    }
    
    caller3()
}

big()
big2()


// only main impacts caller3 