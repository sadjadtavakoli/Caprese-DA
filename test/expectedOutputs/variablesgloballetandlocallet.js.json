{"big-test/unit_tests/variablesgloballetandlocallet.js-3-10":{"impacted":["test/unit_tests/variablesgloballetandlocallet.js"]},"caller1-test/unit_tests/variablesgloballetandlocallet.js-4-7":{"impacted":["big-test/unit_tests/variablesgloballetandlocallet.js-3-10"]},"big2-test/unit_tests/variablesgloballetandlocallet.js-12-20":{"impacted":["test/unit_tests/variablesgloballetandlocallet.js"]},"caller3-test/unit_tests/variablesgloballetandlocallet.js-14-17":{"impacted":["big2-test/unit_tests/variablesgloballetandlocallet.js-12-20","caller3-test/unit_tests/variablesgloballetandlocallet.js-14-17"]},"test/unit_tests/variablesgloballetandlocallet.js":{"impacted":["caller3-test/unit_tests/variablesgloballetandlocallet.js-14-17"]}}